### DISCUSSION TITLE
MTV 80s (Netherlands)

### DISCUSSION THREAD

**Discussion by R4IN2007** [iptv-org/community#782] [Q&A]
### Channel Name

MTV 80s

### Channel ID

MTV80s.uk

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by BellezaEmporium** [comment]
Which country specifically ?

**Comment by R4IN2007** [comment]
> Which country specifically ?

Europe (Netherlands)

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
